c++/test/TestReader.cc

```diff
@@ -218,7 +239,9 @@ namespace orc {
     auto inStream = std::make_unique<MemoryInputStream>(memStream.getData(), memStream.getLength());
     ReaderOptions readerOptions;
     readerOptions.setMemoryPool(*pool);
-    return createReader(std::move(inStream), readerOptions);
+    auto reader = createReader(std::move(inStream), readerOptions);
+    prefetchStripesAndColumnsRandomly(reader.get(), *type);
```

Comment:
Could you write new test cases for prefetching instead of mixing the existing cases? Perhaps it requires some refactoring work to share the common code. In addition, I'd prefer clear parameters instead of (pseudo) random ones.

Reply:
@wgtmac Thanks for your advice. I'll try.